lib/modules/download/generators/gdb.rb

```diff
@@ -70,20 +76,18 @@ def clean_up_after
   end
 
   def clean_up
-    QUERY_CONDITIONS.each do |name, _|
-      FileUtils.rm_rf gdb_component(name)
-    end
+    FileUtils.rm_rf gdb_component
   end
 
   def zip_path
     File.join(@path, "#{@filename}.zip")
   end
 
-  def gdb_component(name)
-    File.join(@path, "#{@filename}_#{name}.gdb")
+  def gdb_component
+    File.join(@path, "#{@filename}.gdb")
```

Comment:
Potentially you could condense  `zip_path` and `gdb_component` into one method to reduce the method count further and then simply pass a variable to determine the file type, what do you think?

Reply:
Yeah that's an option. Although I think I would be more in favour of having 2 different methods as those are essentially two different files even if they share the same name. 
Also `zip_path` is shared across the parent-child structure of the generators and used in the base.rb file (the parent)